activestorage/dummy_data.py

```diff
@@ -112,8 +124,15 @@ def make_holes(var, indices, attribute, value, dummy):
     mindices, findices, vrindices, vm1indices, vm2indices = None, None, None, None, None
     if missing:
         # we use the deprecated missing_value option
-        mindices = [(1,1,1),(n/2,1,1),(1,nm1,1),(nm1,1,n/2)]
-        dvar = make_holes(dvar, mindices, 'missing_value', missing, missing)
+        if pathological:
+            dvar[::2,:,:] = missing
+            dvar.missing_value = missing
+        else:
+            mindices = [(1,1,1),(n/2,1,1),(1,nm1,1),(nm1,1,n/2)]
+            dvar = make_holes(dvar, mindices, 'missing_value', missing, missing)
+
+    if pathological and not missing:
+        raise ValueError('Cannot use pathological keyword without missing keyword')
```

Comment:
ah gotcha, here it is then, well done, but I'd move it up so it fails immediately before doing anything else

Reply:
yep, please do

Reply:
done :+1: